build.savant

```diff
@@ -29,7 +29,7 @@ logbackVersion = "1.4.14"
 slf4jVersion = "2.0.13"
 testngVersion = "7.8.0"
 
-project(group: "org.primeframework", name: "prime-mvc", version: "4.24.0", licenses: ["ApacheV2_0"]) {
+project(group: "org.primeframework", name: "prime-mvc", version: "4.25.0", licenses: ["ApacheV2_0"]) {
```

Comment:
This is probably the most correct version. Since this is only a change to the simulator - we could also consider `4.24.1` since it does not affect runtime compatibility. Eventually, it would be nice to separate the simulator stuff so we don't have to roll the `prime-mvc` version to just pick up changes for test stuff.